### DISCUSSION TITLE
Curious on recommended and minimum requirements for docker

### DISCUSSION THREAD

**Discussion by atechstl** [Budibase/budibase#7910] [General]
Want to run a small app we are working on in Docker in AWS but curious on recommended requirements.  I can either run couch DB or I can connect to RDS database.

**Comment by melohagan** [comment]
Hey @atechstl 

The expected usage will greatly determine the resources required. With that said, we recommend 2 cores and 4GB of RAM as a safe minimum, but you could experiment with less on AWS if you're looking to reduce costs.
